canonical/ubuntu.com
# Case studies templates

## Done

- The base template includes the footer with CTA contact us form:
- Macros for different sections:
  - Hero (copied from Vanilla)
  - Text 
  - Highlights
  - Quotes (copied from Vanilla)
  - Text + divided list
  - Image with caption
- Examples of macros and base template usage are on `templates/case-study/esa-case-study.html`
- Once Vanilla updates their npm package to include macros, the copied over files can be removed.

## QA

- Go to https://ubuntu-com-14142.demos.haus/case-study/
- Check that the example case study matches the design in [Figma](https://www.figma.com/design/AeIJ3GsqnJCHtiBYhkTuTV/24.10-Case-study-template?node-id=0-1&t=sUGdvJIWpE4J9LMw-0)

## Issue / Card

Epic: https://warthogs.atlassian.net/browse/WD-10518

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
